app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/navigation/NavGraph.kt

```diff
@@ -65,7 +56,7 @@ fun NavGraph() {
         }
 
         composable(Routes.ABOUT_SCREEN) {
-            AboutScreen()
+            AboutScreen(popBackStack = { navController.pop() })
```

Comment:
parameter name sounds more descriptive 
```suggestion
            AboutScreen(onBackPressed = { navController.pop() })
```